libr/debug/debug.c

```diff
@@ -1411,6 +1411,7 @@ static int show_syscall(RDebug *dbg, const char *sysreg) {
 	int reg, i, args;
 	RSyscallItem *si;
 	reg = (int)r_debug_reg_get (dbg, sysreg);
+	//todo dont have core here, how to lookup?
 	si = r_syscall_get (dbg->anal->syscall, reg, -1);
```

Comment:
These are the two calls I need to figure out yet. I have dbg here not core. Any ideas how to thread my way from dbg to get the imm value?

Reply:
RDebug have an RCoreBind instance, so you have RCore already, in fact RNum have an RCore instance too

Reply:
we shuold probably add a analOp() method in the RAnalBind/RCoreBind to get this